src/redux/tasks/reducers.ts

```diff
@@ -0,0 +1,88 @@
+import { FETCH_TASK, TASK_ERROR, FETCH_TASK_SUCCESS } from './actions/fetch'
+import { CREATE_TASK_SUCCESS, CREATE_TASK_ERROR } from './actions/create'
+import { UPDATE_TASK_SUCCESS, UPDATE_TASK_ERROR } from './actions/update'
+import { REMOVE_TASK_SUCCESS, REMOVE_TASK_ERROR } from './actions/delete'
+
+import { RootAction } from '../RootAction'
+import { ITask } from '../tasks/types'
+
+export type State = {
+    task: ITask | null,
+    error: string | null,
+    isProcessing: boolean,
+}
+
+const defaultValue: State = {
+    task: null,
+    error: null,
+    isProcessing: false
+}
+export const reducer = (state: State = defaultValue, action: RootAction) => {
+    switch (action.type) {
+
+        case TASK_ERROR:
+            return {
+                ...state,
+                error: action.error,
+                isProcessing: false
+        }
+        
+        case FETCH_TASK:
+            return {
+                ...state,
+                task: null,
+                error: null,
+                isProcessing: true
+        }
+
+        case FETCH_TASK_SUCCESS: 
+            return {
+                ...state,
+                task: action.task,
+                error: null,
+                isProcessing: false
+        }
+
+        case CREATE_TASK_SUCCESS:
+            return {
+                ...state,
+                task: action.task,
+                error: null
+        }
+    
+        case CREATE_TASK_ERROR:
+            return {
+                ...state,
+                error: action.error
+        }
+
+        case UPDATE_TASK_SUCCESS:
+            return {
+                ...state,
+                task: action.task,
+                error: null
+        }
+
+        case UPDATE_TASK_ERROR:
+            return {
+                ...state,
+                error: action.error
+        }
+
+        case REMOVE_TASK_SUCCESS:
+            return {
+                ...state,
+                task: null,
+                error: null
+        }
+
+        case REMOVE_TASK_ERROR:
+            return {
+                ...state,
+                error: action.error
+        }
+
```

Comment:
For Create, Update, and Remove, if we have a problem, this could be great to have a transitional state, for the future.
But it is good without it, for now